case-lib/lib.sh

```diff
@@ -335,7 +335,7 @@ is_sof_used()
 # a wrapper to journalctl with required style
 journalctl_cmd()
 {
-   journalctl -k -q --no-pager --utc --output=short-monotonic \
+   sudo journalctl -k -q --no-pager --utc --output=short-monotonic \
```

Comment:
@aiChaoSONG I just had a look a recent ChromeOS image (less than a couple weeks old) and I found zero trace of `journalctl`, only `rsyslogd`... can you explain?
- https://github.com/thesofproject/sof-test/issues/576#issuecomment-1295641285

Reply:
At the time I tried sof-test on chrome, there was `journalctl`. I have no idea why they remove journalctl in later chrome image, but there is a new tool for log, `craslog`, which works just like `journalctl`.

Reply:
You're right: there was journalctl but only for a short time. With your help I found a bit more info, see #576